### DISCUSSION TITLE
chage color print order with Bambu AMS

### DISCUSSION THREAD

**Discussion by soolang** [OrcaSlicer/OrcaSlicer#5245] [Ideas]
![color](https://github.com/SoftFever/OrcaSlicer/assets/66074699/e67c1987-925a-425e-a2fa-100c1d451cd1)

**i want to print black first, but can not change color print order.
if print whiter first, black line somewhere will on the top of white**

**Comment by superhans65** [comment]
![2024-05-06_08-22](https://github.com/SoftFever/OrcaSlicer/assets/66103310/0c571520-4f7f-433a-8444-138d99daf391)
